### PR TITLE
grammar fix for the description in instruments, misc, clusterbang, and spawnergrenade dm 

### PR DESCRIPTION
**Pull request by RiskySikh**
## About The Pull Request

Like the other grammar fix PR, it simply fixes some grammatical issues in the description for instruments, misc, clusterbang, and spawnergrenade. I know that this PR is really really minor, but I thought it make the descriptions sound better. As well, every minor change makes the game better for everyone.

## Why It's Good For The Game

It just fixes some minor grammatical issues in the description for the four mentioned files above.

## Changelog
:cl:
spellcheck: Fixes some grammatical issues in the code for instruments, misc, clusterbang, and spawnergrenade 
/:cl:

### CODE REVIEW

**Comment by trollbreeder** [comment]
There's a typo in the parrot examine
"the parrot **squaks**, it's a parrot, bawwk!"
would be cool if you fixed that as well

**Comment by RiskySikh** [comment]
> 
> 
> There's a typo in the parrot examine
> "the parrot **squaks**, it's a parrot, bawwk!"
> would be cool if you fixed that as well

fixed it!

File: code/game/objects/items/grenades/clusterbuster.dm [hunk 1]
```diff
@@ -1,191 +1,191 @@
-////////////////////
```
Comment (81Denton) [inline code/game/objects/items/grenades/clusterbuster.dm]: The line endings got fucked up on this file, which editor are you using?

Comment (RiskySikh) [reply code/game/objects/items/grenades/clusterbuster.dm]: > 
> 
> The line endings got fucked up on this file, which editor are you using?

I been using visual studio code, I don't know why my line endings are fucked. I never even touched them, despite that. I was able to run the game fine without any issues.

Comment (spookydonut) [reply code/game/objects/items/grenades/clusterbuster.dm]: either your editor or your git client is forcing line endings to change.

Comment (RiskySikh) [reply code/game/objects/items/grenades/clusterbuster.dm]: I guess so, I mean either way it doesn't screw over the code and I am able to run the game fine.

**Comment by RiskySikh** [comment]
also my travis CI worked successfully for some reason it's been saying pending for 3 days

**Pull request merged**